assets/js/components/Work/Tabs/Structure.jsx

```diff
@@ -25,222 +30,179 @@ const WorkTabsStructure = ({ work }) => {
   const [setWorkImage] = useMutation(SET_WORK_IMAGE, {
     onCompleted({ setWorkImage }) {
       toastWrapper("is-success", "Work image has been updated");
-    }
+    },
   });
 
   if (!work) {
     return null;
   }
 
-  const handleWorkImageChange = id => {
+  const handleDownloadClick = (type) => {
+    console.log("Download clicked and type: ", type);
+  };
+
+  const handleWorkImageChange = (id) => {
     if (id === workImageFilesetId) return;
 
     setWorkImageFilesetId(id === workImageFilesetId ? null : id);
     setWorkImage({ variables: { fileSetId: id, workId: work.id } });
   };
 
-  const onSubmit = data => {
+  const onSubmit = (data) => {
     // TODO : add logic to update filesets for given work.
     console.log(data);
   };
 
   return (
     <form name="work-structure-form" onSubmit={handleSubmit(onSubmit)}>
-      <div className="columns is-centered">
-        <div className="column is-two-thirds box is-shadowless">
-          {work.fileSets.map(({ id, accessionNumber, metadata }) => (
-            <article key={id} className="media">
-              <figure className="media-left">
-                <p className="image is-128x128">
+      <WorkTabsHeader title="Filesets">
+        {!isEditing && (
+          <button
+            type="button"
+            className="button is-primary"
+            onClick={() => setIsEditing(true)}
+          >
+            Edit
+          </button>
+        )}
+        {isEditing && (
+          <>
+            <button type="submit" className="button is-primary">
+              Save
+            </button>
+            <button
+              type="button"
+              className="button is-text"
+              onClick={() => setIsEditing(false)}
+            >
+              Cancel
+            </button>
+          </>
+        )}
+      </WorkTabsHeader>
+
+      <div style={{ marginTop: "1rem" }}>
+        {work.fileSets.map(({ id, accessionNumber, metadata }) => (
+          <article key={id} className="box">
+            <div className="columns">
+              <div className="column is-2">
+                <figure className="image">
```

Comment:
Can you add a class to this image so when the screen becomes smaller, the image size is also reduced ? It is looking rather big compared to the content that is being edited. 
<img width="497" alt="Screen Shot 2020-04-29 at 2 47 45 PM" src="https://user-images.githubusercontent.com/14085957/80641390-6b333900-8a2a-11ea-89fe-dd6766faa8a5.png">

Reply:
For this, mobile views are fairly minimal and how about we wait until someone says something negative about it?  It's kinda tricky to block everything out fullwidth on mobile and have the image look decent without adding extra code bloat (for now) to make it smaller, centered, and such.   Or I'm not sure of a good solution at the current moment.  I agree not ideal, but maybe best case of not that great a selection of cases?